controllers/datafoundation.go

```diff
@@ -2,15 +2,24 @@ package controllers
 
 import (
 	"fmt"
+	"math"
 	"strconv"
 	"strings"
 
 	v1alpha1 "github.com/red-hat-storage/managed-fusion-agent/api/v1alpha1"
+	"github.com/red-hat-storage/managed-fusion-agent/datafoundation"
+	ocsv1 "github.com/red-hat-storage/ocs-operator/api/v1"
 	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/builder"
 )
 
+const (
+	deviceSetName = "default"
```

Comment:
Can we give this a better name? what about `defaultDeviceSetName`? 
To convey meaning and not just what it is

Reply:
Updated